src/lib/features/project/project-service.ts

```diff
@@ -701,8 +702,11 @@ export default class ProjectService {
         );
     }
 
-    private isAdmin(roles: IRoleWithProject[]): boolean {
-        return roles.some((r) => r.name === RoleName.ADMIN);
+    private isAdmin(userId: number, roles: IRoleWithProject[]): boolean {
+        return (
+            userId === SYSTEM_USER_ID ||
```

Comment:
What @thomasheartman mentioned made me think, maybe we should be using ADMIN_TOKEN_USER instead? (-42)

Reply:
What problem does that solve?